src/prelude.rs

```diff
@@ -33,9 +33,11 @@ pub mod tcp {
 pub use crate::server;
 
 ///////////////////////////////////////////////////////////////////
+#[cfg(feature = "server")]
```

Comment:
fuck... that's rustfmd doing it for who knows why... I even disabled it in my editor to update it but it seems it still found a way.

For the exports, actually, since in the interface, `OptionalResponsePdu` is only used as an `Into<>` bound, it is not required to export its inner ResponsePdu, since Using a `Response` or `Option<Response>` in the actual Services works.

And yes, I'm removing it from prelude. I think I had a clippy warning at some point about it not requiring to be pub because it was not exported that made me put it in prelude but I cant see it anymore... I'm completely removing it from prelude :+1: